tests/test_entries.py

```diff
@@ -0,0 +1,139 @@
+        self.assertIn('At Russia', str(req_all.data))
+
+    def test_entries_contains_nothing(self):
+        """ Test fetch all entries """
+        req_all = self.client().get(self.entry_route)
+        self.assertEqual(req_all.status_code, 200)
+
+    def test_entry_creation(self):
+        """Test entry creation via post method"""
+        # bind the app to the current context
+        with self.app.app_context():
+            req = self.client().post(self.entry_route, data=self.entry)
+            req2 = self.client().post(self.entry_route, data=self.entry_bad_title)
+            req3 = self.client().post(self.entry_route, data=self.entry_no_description)
+            req4 = self.client().post(self.entry_route, data=self.entry_no_title)
+            self.assertEqual(req2.status_code, 401)
+            self.assertEqual(req3.status_code, 401)
+            self.assertEqual(req4.status_code, 401)
+            self.assertEqual(req.status_code, 201)
+            self.assertIn('At Russia', str(req.data))
+
+    def test_landing_page_message(self):
+        """ Test Landing page message"""
+        req = self.client().get('/api/v1/')
+        self.assertEqual(req.status_code, 200)
+
+    def test_fetch_single_entry(self):
+        """ Test fetch single entry """
+        # Not neccessary to use the following variable so pylint unused variable warning is disabled
+        req = self.client().post(self.entry_route, data=self.entry) # pylint: disable=unused-variable
+        req_single = self.client().get(self.single_entry_route)
+        self.assertEqual(req_single.status_code, 200)
+        self.assertIn('At Russia', str(req_single.data))
+
+    def test_modify_single_entry(self):
+        """ Test editing of single entry """
+         # Not neccessary to use the following variable so pylint unused variable warning is disabled
+        req = self.client().post(self.entry_route, data=self.entry) # pylint: disable=unused-variable
+        req_single = self.client().put(self.single_entry_route, data=self.entry_new)
+        req2 = self.client().put(self.single_entry_route, data=self.entry_bad_title)
+        req3 = self.client().put(self.single_entry_route, data=self.entry_no_description)
+        req4 = self.client().put(self.single_entry_route, data=self.entry_no_title)
+        self.assertEqual(req2.status_code, 401)
+        self.assertEqual(req3.status_code, 401)
+        self.assertEqual(req4.status_code, 401)
+        self.assertEqual(req_single.status_code, 201)
+        self.assertIn('At Beach', str(req_single.data))
+
+    def test_not_found_url(self):
+        """ Test for unavailable url request """
+        req = self.client().get(self.bad_url)
+        self.assertEqual(req.status_code, 404)
+
+    def test_error_405(self):
+        """ Test for Method Not allowed """
+        req = self.client().post(self.single_entry_route)
+        self.assertEqual(req.status_code, 405)
+
+class TestDeletion(unittest.TestCase):
```

Comment:
You do not need a whole class to test deletion. It's not OOP because its not a unit of anything. Just add the delete tests in the existing class.

Reply:
Alright. Thanks for the feedback, I will rectify that.